### PR TITLE
build: update to latest @angular/dev-infra-private and husky@5

### PR DESCRIPTION
**Pull request by josephperrott**
This is the patch only change for #21080

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -340,6 +340,7 @@
 
 # Misc
 /.github/**                                        @angular/dev-infra-components
+/.husky/**                                        @angular/dev-infra-components
```
Comment (crisbeto) [inline .github/CODEOWNERS:343]: nit: the indentation is slightly off here.

**Review by crisbeto — APPROVED** [review]
LGTM

**Comment by mmalerba** [comment]
@josephperrott looks like CI is still failing

**Comment by josephperrott** [comment]
@mmalerba The CI failures are upstream: https://app.circleci.com/pipelines/github/angular/components?branch=11.0.x

**Comment by mmalerba** [comment]
whoops, should be fixed once you rebase

**Comment by josephperrott** [comment]
@mmalerba rebased

**Pull request merged**